BETA: Register dtech.is-a.dev

pgwiz opened the pull request:
Added `dtech.is-a.dev` using the [dashboard](https://manage.is-a.dev).

pgwiz committed:
feat(domain): dtech.is-a.dev

orangci committed:
Merge branch 'main' into main

orangci reviewed (changes requested):
Your CNAME should point somewhere, not be the domain you want.

pgwiz committed:
Update dtech.json

pgwiz reviewed (commented):
Changed record to TXT

CuteDog5695 commented on domains/dtech.json:
```diff
@@ -4,6 +4,6 @@
     },
     "record": {
-        "CNAME": "dtech.is-a.dev"
+        "TXT": "_github-pages-challenge-pgwiz.dtech"
```
yea no this wont work, refer to here on how to connect a github page https://www.is-a.dev/docs/github_pages/

pgwiz committed:
Update dtech.json

pgwiz reviewed (commented):
What if i temporarily connect it to my panel at that IP address

CuteDog5695 commented on domains/dtech.json:
```diff
@@ -4,6 +4,6 @@
     },
     "record": {
-        "TXT": "_github-pages-challenge-pgwiz.dtech"
+        "A": "128.204.223.70"
```
IP needs to be in an array

CuteDog5695 replied:
also gives error, probably because it isnt connect to a domain. if you want to connect it there then please give us a preview of your website

pgwiz committed:
Update dtech.json

My panel has only 1 ip already linked it

My website ponting to the same ip can be seen at https://wiptechg.69.mu

pgwiz commented:
I

pgwiz commented:
1. That is the dns zone settings on my panel automatically created.

![Screenshot_20240503_071441_WhatsApp](https://github.com/is-a-dev/register/assets/75813303/2d4bd965-5d7d-4537-b60f-0c0bcdde7f58)


2. And the website preview at https://wiptechg.69.mu 

![Screenshot_20240503_204815_Chrome](https://github.com/is-a-dev/register/assets/75813303/649d9d1b-2e5d-429d-9f80-08948b0fa10e)

orangci committed:
Merge branch 'main' into main

orangci reviewed (changes requested):
Your site is inaccessible. "Error: unknown domain"
And your preview is incomplete.

orangci committed:
Merge branch 'main' into main

orxngc commented:
Stale
